cidc_api/models/files/facets.py

```diff
@@ -200,6 +200,11 @@ def __init__(self, match_clauses: List[str], description: Optional[str] = None):
         "Images": FacetConfig(["/ihc/ihc_image."]),
         "Combined Markers": FacetConfig(["csv|ihc marker combined"]),
     },
+    "TCR": {
+        "Source": FacetConfig(
+            ["/tcr/r1.fastq.gz", "/tcr/r2.fastq.gz", "/tcr/i1.fastq.gz", "/tcr/i2.fastq.gz"]
+        ),
```

Comment:
Not sure whether the sample sheet should be accessible by the file browser, hence not adding a facet group atm